draft-ietf-quic-transport.md

```diff
@@ -2031,8 +2031,8 @@ this value differs from previous packets.  Ths results in two problems:
   about this.
 
 The Packet Number field is set to a randomized value.  The server SHOULD send a
-packet with a short header and a type of 0x1F.  This produces the shortest
-possible packet number encoding, which minimizes the perceived gap between the
+packet with a short header and a packet number length of 1 octet. Using the
+shortest possible packet number encoding minimizes the perceived gap between the
 last packet that the server sent and this packet.  A server MAY use a different
 short header type, indicating a different packet number length, but a longer
```

Comment:
You might want to make a similar modification down here, "A server MAY indicate...."